test_floor.k

```diff
@@ -0,0 +1,6 @@
+import math
+
+a = 2.4
+a_floor = math.floor(a)
+a_ceil = math.ceil(a)
+
```

Comment:
Please put this test into https://github.com/kcl-lang/kcl/tree/main/test/grammar/builtins/math/floor

Reply:
It's mean you need to add the testcase into https://github.com/kcl-lang/kcl/blob/main/test/grammar/builtins/math/floor/main.k and add the expected text into  https://github.com/kcl-lang/kcl/blob/main/test/grammar/builtins/math/floor/stdout.golden
@Transcendental-Programmer

Reply:
```diff
import math

a = math.floor(-45.17)
b = math.floor(100.12)
++ c = 2.4
++ c_floor = math.floor(c)
++ c_ceil = math.ceil(c)
```